### DISCUSSION TITLE
How to Retrieve Available F1 Seasons from the API?

### DISCUSSION THREAD

**Discussion by dalton-f** [br-g/openf1#102] [Q&A] [answered]
Hi everyone,

I’m working on a website that lets users select a specific Formula 1 season and compare two drivers from that season at a particular race event. I’m looking for a straightforward way to directly call the API to get a list of seasons that have data available. It seems like there isn’t a direct endpoint for this, and I’d prefer not to filter the years from session data manually.

Does anyone know of an easy method or endpoint to get a list of available seasons directly? Any advice or workarounds would be greatly appreciated!

Thanks!

**Comment by guid118** [comment, marked as answer]
using the meetings endpoint you can get all race weekends (https://api.openf1.org/v1/meetings)
this way you can look through a weekend at a time. Currently it seems like there is only 40 recorded sessions, since the start of the 2023 season.
